packages/sitecore-jss-nextjs/src/components/Link.tsx

```diff
@@ -35,6 +37,11 @@ export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
       return null;
     }
 
+    // when metadata is present, render it to be used for chrome hydration
+    if (metadata) {
```

Comment:
We need to continue to respect the (user-defined) `editable` prop for all of these
```suggestion
    if (metadata && editable) {
```